package.json

```diff
@@ -7,12 +7,15 @@
     "npm": ">=6.4.1"
   },
   "dependencies": {
+    "@date-io/date-fns": "1.3.13",
     "@material-ui/core": "^4.10.1",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/pickers": "3.2.10",
     "@reduxjs/toolkit": "^1.2.3",
     "@testing-library/jest-dom": "^5.11.0",
     "axios": "^0.19.2",
     "clsx": "^1.1.1",
+    "date-fns": "2.16.1",
```

Comment:
Do we need another date package? As we formatted the date using `moment` in some other place.